poem/Cargo.toml

```diff
@@ -131,7 +131,9 @@ libcookie = { package = "cookie", version = "0.18", features = [
     "secure",
 ], optional = true }
 opentelemetry-http = { version = "0.27.0", optional = true }
-opentelemetry-semantic-conventions = { version = "0.16.0", optional = true }
+opentelemetry-semantic-conventions = { version = "0.27.0", optional = true, features = [
+    "semconv_experimental"
```

Comment:
needed for https://github.com/poem-web/poem/blob/6f44a889100a595df33fa40ec337edc961ab83c4/poem/src/middleware/opentelemetry_tracing.rs#L136

https://github.com/poem-web/poem/actions/runs/12008274868/job/33470583472?pr=916#step:9:386

<img width="1058" alt="image" src="https://github.com/user-attachments/assets/d364ae69-9540-4a55-9ba3-5fbd7ae3dc92">